pkg/scheduler/helper.go

```diff
@@ -18,15 +18,15 @@
 
 package scheduler
 
-import "github.com/apache/incubator-yunikorn-core/pkg/api"
+import "github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/common"
```

Comment:
This whole file should be removed as it is not used.

Reply:
Indeed. Deleted the whole file.